src/Table/Footer.less

```diff
@@ -9,7 +9,7 @@
 }
 
 .Table--footer--cell {
-  font-size: @size_s;
+  font-size: @size_m;
```

Comment:
Maybe use the type size utilities instead here: https://github.com/Clever/components/blob/master/src/less/type-size.less